fuzz/targets/mod.rs

```diff
@@ -197,3 +197,33 @@ pub fn fuzz_coprocessor_codec_time_from_u64(data: &[u8]) -> Result<(), Error> {
     let t = Time::from_packed_u64(u, time_type, fsp, &tz)?;
     fuzz_time(t, cursor)
 }
+
+// Duration
+fn fuzz_duration(
+    t: tikv::coprocessor::codec::mysql::Duration,
+    mut cursor: Cursor<&[u8]>,
+) -> Result<(), Error> {
+    use tikv::coprocessor::codec::mysql::DurationEncoder;
+    let _ = t.fsp();
+    let _ = t.clone().set_fsp(cursor.read_as_u8()?);
```

Comment:
I suppose the error caused by `read_as_u8` should be handled elsewhere (so we ensure all these fuzzing codes are executed).

Reply:
This line of code is just copied from existing fuzz tests above. I believe that it is not hurtful since  the fuzz testing will take huge of iterations.